### PR TITLE
Fixed link path "apis.md"

### PR DESCRIPTION
**Pull request by PINTO0309**
Improvement of link disconnection by document movement.

### CODE REVIEW

File: tensorflow/lite/g3doc/apis.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ convenience.  Similarly, consistency with TensorFlow APIs was not an explicit
 goal and some variance is to be expected.
 
 There is also a Python API for TensorFlow Lite described
-[here](../toco/g3doc/python_api.md#interpreter).
```
Comment (gargn) [inline tensorflow/lite/g3doc/apis.md]: Given that this is a document about the TFLite interpreter APIs, `#interpreter` seems correct.

To clarify, can you change "TensorFlow Lite described" --> "TensorFlow Lite interpreter, described".

Thanks for making this fix.

**Review by gargn — CHANGES_REQUESTED** [review]

File: tensorflow/lite/g3doc/apis.md (hunk 1 above)
Comment (PINTO0309) [reply tensorflow/lite/g3doc/apis.md]: @gargn 
Thank you for review.
I think you are right.
I fixed the referenced anchor.
Can you please review again?
**From**
```
#converter-python-api-guide
```
**To**
```
#tensorflow-lite-python-interpreter-
```

Comment (gargn) [reply tensorflow/lite/g3doc/apis.md]: To clarify what I meant, can you make your change the following:

There is also a Python API for TensorFlow Lite interpreter described [here](convert/python_api.md#interpreter).

**Review by gargn — CHANGES_REQUESTED** [review]

File: tensorflow/lite/g3doc/apis.md (hunk 1 above)
Comment (PINTO0309) [reply tensorflow/lite/g3doc/apis.md]: @gargn 
Sorry for the inconvenience to you.
Your suggestion:
https://github.com/tensorflow/tensorflow/pull/convert/python_api.md#interpreter
bbb
And, And in your previous proposal **`../toco/g3doc/python_api.md`** there is no document in the current **`master`** branch.
It appears to have been deleted 3 days ago.
aaa
And the current **`tensorflow/lite/g3doc/convert/python_api.md`** does not have "#interpreter" anchor.
I wonder how to use Github. . .
I am a little confused. . .

**Comment by tensorflowbutler** [comment]
Nagging Assignee @wt-huang: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request closed without merging**